frontend/src/helpers/ForestClientUserSession.ts

```diff
@@ -84,7 +84,7 @@ class ForestClientUserSession implements SessionProperties {
         name: toTitleCase(parsedUser["custom:idp_display_name"]),
         provider: provider,
         userId: `${provider}\\${parsedUser["custom:idp_username"] ?? parsedUser["custom:idp_user_id"]}`,
-        birthDate: parsedUser["birthdate"],
+        birthdate: parsedUser["birthdate"],
```

Comment:
Use `java.time.LocalDate` instead of `String`

Reply:
This is TypeScript :-)